app/assets/javascripts/views/common/tableView.js

```diff
@@ -0,0 +1,174 @@
+(function(App) {
+
+  'use strict';
+
+
+  App.View.TableView = Backbone.View.extend({
+
+    defaults: {
+      resultsPerPage: 3
+    },
+
+    el: '#table',
```

Comment:
We could better let the parent view decide of the `el` element instead of using `#table`.